src/foundry/common/prosemirror/dirty-plugin.d.mts

```diff
@@ -1,13 +1,13 @@
 import type { Schema } from "prosemirror-model";
 import type { Plugin } from "prosemirror-state";
 import type ProseMirrorPlugin from "./plugin.d.mts";
-import type { EmptyObject } from "../../types/utils.d.mts";
+import type { EmptyObject } from "../../../types/utils.d.mts";
 
 export default ProseMirrorDirtyPlugin;
 
 /**
  * A simple plugin that records the dirty state of the editor.
  */
 declare class ProseMirrorDirtyPlugin extends ProseMirrorPlugin {
-  static override build(schema: Schema, options?: EmptyObject): Plugin;
+  static override build(schema: Schema, options?: EmptyObject | undefined): Plugin;
```

Comment:
So `| undefined` in an optional function parameter has no effect. The only difference is with object properties. So you don't need `| undefined` here or in other function parameters.

Reply:
I will remember this eventually